modules/wallet/persist.go

```diff
@@ -71,6 +74,17 @@ func (w *Wallet) openDB(filename string) (err error) {
 			wb.Put(keySiafundPool, encoding.Marshal(types.ZeroCurrency))
 		}
 
+		// build the bucketAddrTransactions bucket if necessary
+		if buildAddrTxns {
+			it := dbProcessedTransactionsIterator(tx)
+			for it.next() {
+				index, pt := it.key(), it.value()
+				if err := dbAddProcessedTransactionAddrs(tx, pt, index); err != nil {
+					return err
+				}
+			}
+		}
```

Comment:
not sure how long this would take on a large wallet. It probably won't be noticeable to anyone with less than 10k transactions. We should include a warning that upgrading the wallet may be slow if you have lots of transactions.

Reply:
That's okay. People with that many transactions will be used to it being slow